app/models/user.rb

```diff
@@ -3,4 +3,14 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+
+  has_many :issues, dependent: :destroy
+  has_many :bookmarks, dependent: :destroy
+  has_many :comments, dependent: :destroy
+  has_many :issue_tags, through: :issues
+  has_many :tags, through: :issue_tags
```

Comment:
I don't remember, why do we consider that a user has many tags/issue_tags? How are we using this?